strict-version-matcher-plugin/build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
     testCompile 'junit:junit:4.12'
     testCompile "com.google.truth:truth:0.39"
     implementation 'com.google.code.findbugs:jsr305:3.0.0'
+    compile 'com.google.guava:guava:23.4-android'
```

Comment:
Should we be using the Android variant or the normal one since the runtime here is build tooling.